tracee-rules/engine/engine.go

```diff
@@ -152,11 +154,31 @@ func (engine *Engine) consumeSources(done <-chan bool) {
 					engine.signaturesMutex.RUnlock()
 					continue
 				}
+
+				u, err := traceeEvt.ToUnstructured()
+				if err != nil {
+					engine.logger.Printf("unstructuring tracee event: %v", err)
+					continue
+				}
```

Comment:
Since this will be throwaway code post the update to v0.30.0, I think we can forgo writing a test here.

Reply:
💯